src/components/MapAndTableControls/components/ViewToggle.jsx

```diff
@@ -76,6 +76,18 @@ const ViewToggle = ({ view, setView }) => {
     csvLinkRef.current.link.click()
   }
 
+  const downloadedFileName = () => {
+    const getFormattedDate = () => {
+      const today = new Date()
+      const year = today.getFullYear()
+      const month = String(today.getMonth() + 1).padStart(2, '0')
+      const day = String(today.getDate()).padStart(2, '0')
+      return `${year}-${month}-${day}`
```

Comment:
This appears to work, but have you considered making use of `new Date.toISOString()` which would result in something like `'2024-07-26T23:28:03.827Z'`. Then you can remove everything from the T onwards.

Something like this:
``` js
const formattedDate = new Date().toISOString().slice(0,10)
/* = '2024-07-26' */
```